### PR TITLE
Integrating box update into pristine

### PR DESCRIPTION
**Pull request by syntacticvexation**

### CODE REVIEW

**Comment by fgrehm** [comment]
Hey @syntacticvexation , sorry for the huge delay but are u still interested in the project? I haven't used it for more than a year now. LMK if u want to keep maintaining it and I can give u write permissions to the project.

**Comment by syntacticvexation** [comment]
Hi @fgrehm,

More than happy to do that, I still find it really useful. :)

Cheers.

**Comment by fgrehm** [comment]
Invited you to the project, LMK when u are ready to cut a release :D

File: lib/vagrant-pristine/plugin.rb [hunk 1]
```diff
@@ -59,10 +61,42 @@ def execute
         # Success if no confirms were declined
         return 1 if declined
 
+        with_target_vms(argv, :provider => options[:provider]) do |machine|
```
Comment (fgrehm) [inline lib/vagrant-pristine/plugin.rb]: Did u mean to add some code to this block?

**Comment by syntacticvexation** [comment]
I've cleaned that up and added some flags to handle the update behaviour, I think it's ready to release. :)

**Comment by fgrehm** [comment]
Hey @syntacticvexation, please send me the email you registered on rubygems and I'll add you there. Tks!

**Comment by syntacticvexation** [comment]
I've sent you an email, it's the same username - syntacticvexation.

**Comment by syntacticvexation** [comment]
I've sent you an email, it's the same username - syntacticvexation.

**Comment by fgrehm** [comment]
Done!

**Pull request merged**